spec/unit/hanami/events/matcher_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe Hanami::Events::Matcher do
+  let(:matcher) { described_class.new(pattern) }
+
+  context 'when pattern match only one event' do
+    let(:pattern) { 'user.created' }
+
+    it { expect(matcher.match?('user.created')).to be true }
```

Comment:
Shared Examples is a common way to get rid from duplication in RSpec: http://www.betterspecs.org/#sharedexamples
This level of abstraction allows us to focus on behavior ('match/no match') rather than on code related to implementation.
If implementation changes we will need to change just 2 lines of code instead of 60 lines with current code.